### PR TITLE
Redirect if URL is not appr.tc and redirect is not handled by ISP

### LINKED ISSUE #221: www.appr.tc, apprtc.net, should redirect to appr.tc
Otherwise, they will have different permissions, cookies, etc.

### PR DESCRIPTION
**Pull request by KaptenJansson**
**Description**
Redirect old URL's not handled by the ISP to appr.tc and make sure the path and URL paramters tag along.

Fixes #221

**Purpose**
Make everyone use appr.tc.

### CODE REVIEW

**Comment by samdutton** [comment]
I'm not a Python boffin, but LGTM.

File: src/app_engine/apprtc.py [hunk 1]
```diff
@@ -590,6 +590,20 @@ def get(self):
     params = get_room_parameters(self.request, None, None, None)
     self.response.write(json.dumps(params))
 
+def checkIfRedirect(self):
+  REDIRECT_DOMAINS = [
```
Comment (hkjellander) [inline src/app_engine/apprtc.py]: Would it make sense to move this into app_engine/constants.py even if it's only used here. Buried 600 lines down in source code is a pretty hidden place.

File: src/app_engine/apprtc.py [hunk 2]
```diff
@@ -590,6 +590,20 @@ def get(self):
...
+    'apprtc.appspot.com', 'apprtc.webrtc.org', 'www.appr.tc'
+  ]
+  ARGUMENTS = self.request.arguments()
+  parsed_args = ''
+  if self.request.headers['Host'] in REDIRECT_DOMAINS:
+    webapp2.redirect('https://appr.tc', permanent=True)
```
Comment (hkjellander) [inline src/app_engine/apprtc.py]: Why do a redirect here and then another one at line 606? It seems the one at line 606 is the correct one and that this one should be removed. Or have a I misunderstood how webapp2.redirect works? I tried finding documentation but couldn't find anything pointing at any reason using it like this.

File: src/app_engine/apprtc.py [hunk 3]
```diff
@@ -590,6 +590,20 @@ def get(self):
     params = get_room_parameters(self.request, None, None, None)
     self.response.write(json.dumps(params))
 
+def checkIfRedirect(self):
+  REDIRECT_DOMAINS = [
+    'apprtc.appspot.com', 'apprtc.webrtc.org', 'www.appr.tc'
+  ]
+  ARGUMENTS = self.request.arguments()
```
Comment (hkjellander) [inline src/app_engine/apprtc.py]: The ARGUMENTS constant doesn't add anything useful, just use self.request.arguments() directly at line 601 instead.

File: src/app_engine/apprtc.py [hunk 4]
```diff
@@ -590,6 +590,20 @@ def get(self):
...
+  parsed_args = ''
+  if self.request.headers['Host'] in REDIRECT_DOMAINS:
+    webapp2.redirect('https://appr.tc', permanent=True)
+    for argument in ARGUMENTS:
+      if parsed_args == '':
+        parsed_args += '?'
+      parsed_args += argument + '&'
+    redirect_url = 'https://appr.tc' + self.request.path + parsed_args
```
Comment (hkjellander) [inline src/app_engine/apprtc.py]: Move 'https://appr.tc' into a constant in app_engine/constants.py

File: src/app_engine/apprtc.py [hunk 5]
```diff
@@ -590,6 +590,20 @@ def get(self):
     params = get_room_parameters(self.request, None, None, None)
     self.response.write(json.dumps(params))
 
+def checkIfRedirect(self):
+  REDIRECT_DOMAINS = [
+    'apprtc.appspot.com', 'apprtc.webrtc.org', 'www.appr.tc'
```
Comment (hkjellander) [inline src/app_engine/apprtc.py]: Out of curiosity I also tried www.apprtc.appspot.com and it gives me an "invalid certificate" error. Is that something we want to fix?

File: src/app_engine/apprtc.py (hunk 3 above)
Comment (KaptenJansson) [inline src/app_engine/apprtc.py]: Done

File: src/app_engine/apprtc.py (hunk 1 above)
Comment (KaptenJansson) [inline src/app_engine/apprtc.py]: Done

File: src/app_engine/apprtc.py (hunk 5 above)
Comment (KaptenJansson) [inline src/app_engine/apprtc.py]: I guess we forgot to include www. in the certificate. Not sure if this is important enough to warrant us to create a new certificate. @juberti @samdutton  WDYT?

**Pull request merged**